cmd/consumer/responses.go

```diff
@@ -297,6 +297,10 @@ func injectMessageResponses(ma *handler.MessageActions) {
 		`Don't ask to ask. Just ask. We'll let you know if there's a better place to ask.`,
 		`- <https://dontasktoask.com/>`,
 	)
+
+	ma.HandleStatic("crosspost", "cross-posting to multiple channels", []string{"crosspost", "xpost"},
```

Comment:
Could you remove `crosspost` from the aliases `[]string`? They are meant to be triggers in addition to the primary one.

Reply:
Done!